data/en/items/imagemanagement.json

```diff
@@ -0,0 +1,37 @@
+{
+      "title": "Quarantine of Docker Images in Docker Registries that have discovered issues",
+      "priority": "High",
+      "description": "Use policy to protect images from drift while in the registry, on both push and pull.",
+      "detail": "On build the image is secured based on the threshold set, but now while in the registry a new issue is discoverd.  You need to ensure that the image can not be deployed until the issue is remediated.",
+      "documentation": [
+        {
+          "title": "ACR Quarantine",
+          "url": "https://github.com/Azure/acr/tree/master/docs/preview/quarantine"
+        }
+      ],
+      "tags": ["all", "security"]
+    },
+    {
+      "title": "Role-Based Access Contol (RBAC) to Docker Registries",
+      "priority": "High",
+      "description": "The Azure Container Registry service supports a set of built-in Azure roles that provide different levels of permissions to an Azure container registry. Use Azure role-based access control (RBAC) to assign specific permissions to users, service principals, or other identities that need to interact with a registry.",
+      "documentation": [
+        {
+          "title": "Azure Container Registry roles and permissions",
+          "url": "https://docs.microsoft.com/en-us/azure/container-registry/container-registry-roles"
+        }
+      ],
+      "tags": ["all", "security"]
+    },
+    {
+      "title": "Network Segmentation of Docker Registries",
+      "priority": "High",
+      "description": "Limit access to a registry by assigning virtual network private IP addresses to the registry endpoints and using Azure Private Link. Network traffic between the clients on the virtual network and the registry's private endpoints traverses the virtual network and a private link on the Microsoft backbone network, eliminating exposure from the public internet. Private Link also enables enables private registry access from on-premises through Azure ExpressRoute private peering or a VPN gateway.",
+      "documentation": [
+        {
+          "title": "Azure Container Registry Private Link",
+          "url": "https://docs.microsoft.com/en-us/azure/container-registry/container-registry-private-link"
+        }
+      ],
+      "tags": ["all", "network", "security"]
+    },
```

Comment:
@miwithro I don't understand this file. it's not even a parsable json.

and I don't see the difference between this PR and the previous one, it seems you forgot to update your branch before pushing again, I have to compare line by line :-/

Reply:
Hmm, I didn't update management.json. Investigating.